
# Wrap the rtabmap::SMState class

Header header

uint32 sensorStep
float32[] sensors

uint32 actuatorStep
float32[] actuators

# Optional fields
sensor_msgs/Image image
rtabmap/KeyPoint[] keypoints
